modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/test/ContentDashboardAdminPortletTest.java

```diff
@@ -237,6 +237,39 @@ public void testGetAuditGraphTitleWithMissingChildAssetVocabulary()
 		}
 	}
 
+	@Test
+	public void testGetContextWithLtrLanguageDirection() throws Exception {
+		MockLiferayPortletRenderRequest mockLiferayPortletRenderRequest =
+			_getMockLiferayPortletRenderRequest();
+
+		mockLiferayPortletRenderRequest.setAttribute(
```

Comment:
As many of the tests has the COMPANY_ID attribute, I think that would be nicer to add it to the _getMockLiferayPortletRenderRequest() method directly, as it takes _company.getCompanyId()